source/postprocess/composition_limiter_statistics.cc

```diff
@@ -0,0 +1,248 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#include <aspect/postprocess/composition_BPL_statistics.h>
+#include <aspect/postprocess/composition_statistics.h>
+#include <aspect/simulator_access.h>
```

Comment:
I just saw that this is probably copy-paste from composition_statistics.cc, where simulator_access.h is included unnecessarily as well. Feel free to propose another PR fixing that one. Otherwise I will do it next week.